File: spigot/src/test/java/kr/entree/samplespigot/SpigradleSampleTest.java
package kr.entree.samplespigot;

import be.seeseemelk.mockbukkit.MockBukkit;
import be.seeseemelk.mockbukkit.ServerMock;
import org.junit.Test;

/**
 * Created by JunHyung Lim on 2020-06-20
 */
public class SpigradleSampleTest {
    @Test
    public void test() {
        ServerMock serverMock = MockBukkit.mock();
        SpigradleSample mockPlugin = MockBukkit.load(SpigradleSample.class);
        mockPlugin.onEnable();
    }
}

File: spigot-kotlin/src/test/kotlin/kr/entree/samplespigot/SpigradleSampleTest.kt
package kr.entree.samplespigot

import be.seeseemelk.mockbukkit.MockBukkit
import org.junit.Test

/**
 * Created by JunHyung Lim on 2020-06-24
 */
class SpigradleSampleTest {
    @Test
    fun test() {
        val serverMock = MockBukkit.mock()
        val mockPlugin = MockBukkit.load(SpigradleSample::class.java)
        mockPlugin.onEnable()
    }
}